src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+
+![Printer Settings Screen](/images/printers/printer-settings.png)
+
+---
+
+## CUPS and Printer Compatibility
+
+Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, print queues, and recognizes connected printers. While CUPS recognizes a wide range of printer models, printer functionality may vary. Check if your printer is compatible using [OpenPrinter.com](https://www.openprinting.org/printers).
+
+## Checking Printer Status in CUPS
+
+Open this web page to look at the CUPS (Common Unix Printing System) configuration and status window:
+
+[localhost:631](http://localhost:631)
+
+The status window will show current print jobs, detected printers, and other information about the printing system. If you would like to share this printer with others on your local network, click on the 'Admin' link, under Server, click on the "Share printers connected to this system" and save the changes. Other computers on your network should than see that printer. When there is a prompt for your username and password, use your user name, and password used to login.
+
+![CUPS Status Window](/images/printers/cups-settings.png)
+
+CUPS allows you to manage printers using Terminal commands. See the [CUPS documentation](https://www.cups.org/doc/admin.html) for more information.
+
+## Adding a Printer
+
+Most printers will be automatically added to the computer. If a printer is not automatically added, you can manually add printers in `Settings` ➞ `Printers`.
+
+1. Press the `Super` key and type the word Printers.
+
+2. Choose the `Printers` application in the search box.
+
+3. In the `Printers Application`, click the `Add a Printer...` button and a box will pop up with different options.
+
+    ![Click Add a Printer](/images/printers/add-a-printer.png)
+
+4. Wait a few seconds for printers to appear in the Device List.
+
+5. Select your printer and click `Add`.
+
+    ![Click Add](/images/printers/click-add.png)
+
+## HP Printers
+
+HP printers are supported with the hplip package, which is installed by default in Pop!\_OS.
+
+```bash
+sudo apt install hplip
+```
+
+### HP Device Manager (GUI)
+
+If you would like to use a guided GUI application from HP, you will need to install a python dependency:
+
+```bash
+sudo apt install python3-pyqt5
+```
+
+Then run `hp-setup` to start the HP Device Manager.
+
+![HP Device Manager](/images/printers/hp-setup.png)
+
+## Brother Printers
+
+Brother provides a driver installation tool for Linux users. Install the appropriate driver for your Brother printer by downloading this tool and running the installer with your printer model appended to the command.
+
+1. Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to download Brother's Driver Install Tool for Linux while searching for the appropriate printer.
```

Comment:
> Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to download Brother's Driver Install Tool for Linux while searching for the appropriate printer.

You're not doing these things at the same time; this seems like an incorrect use of the word "while." You probably want to resequence so the instructions are in the correct order:

> Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) to search for the appropriate printer and download Brother's Driver Install Tool for Linux.

Or better yet, since downloading seems to be redundant with the next step, don't say it twice:

> Click [this link](https://support.brother.com/g/b/productsearch.aspx?c=us&lang=en&content=dl) and search for the appropriate printer.